app/constants.php

```diff
@@ -81,6 +81,7 @@
 defined('LDAP_USER_ATTRIBUTE_PHOTO') or define('LDAP_USER_ATTRIBUTE_PHOTO', '');
 defined('LDAP_USER_ATTRIBUTE_LANGUAGE') or define('LDAP_USER_ATTRIBUTE_LANGUAGE', '');
 defined('LDAP_USER_CREATION') or define('LDAP_USER_CREATION', true);
+define('LDAP_USER_DEFAULT_ROLE_MANAGER') or define('LDAP_USER_DEFAULT_ROLE_MANAGER', false);
```

Comment:
There is a mistake here, the first part should be `defined`. You can see this error from the unit tests: `Warning: define() expects at least 2 parameters, 1 given in /home/travis/build/kanboard/kanboard/app/constants.php on line 84`

Reply:
Yes